src/environments/environment.ts

```diff
@@ -1,6 +1,7 @@
 
 export const environment = {
   production: false,
+  appVersion: require('../../package.json').version + '-dev',
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/personalmoney/app-supabase/pullRequest?prid=7587366)
```suggestion
  appVersion: require("../../package.json").version + '-dev',
```